server/common/pom.xml

```diff
@@ -43,6 +43,22 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
+    <name>Broadcom COBOL LS Common Module</name>
+    <description>Provides dialect's support for 3rd party companies</description>
+    <url>https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support</url>
```

Comment:
@zimlu02 please take a look on url tag. It should be an url to the project. Now it is COBOL LS Marketplace url

Reply:
Provides dialect support for third party companies